sim/pokemon.ts

```diff
@@ -1174,7 +1174,15 @@ export class Pokemon {
 		}
 		let boostName: BoostID;
 		for (boostName in pokemon.boosts) {
-			this.boosts[boostName] = pokemon.boosts[boostName]!;
+			this.boosts[boostName] = pokemon.boosts[boostName];
+			if (this.battle.gen <= 1) {
```

Comment:
I would just do equals gen 1, there's nothing prior to that.

Reply:
there was other code in this file that does `<= 1` which is why I did that (`=== 1` and `<= 1` are used pretty interchangeably throughout the entire codebase though so 🤷‍♂️)

Reply:
Idk why we'd do it that way, but I suppose that can be a thing for another day.